packages/cli/src/models/dependency/Dependency.ts

```diff
@@ -59,17 +63,21 @@ export default class Dependency {
 
   public static hasDependenciesForDeploy(network: string): boolean {
     const dependencies = ProjectFile.getLinkedDependencies() || [];
-    const networkDependencies =
-      NetworkFile.getDependencies(`zos.${network}.json`) || {};
```

Comment:
We can remove the `getDependencies` static method at `NetworkFile.ts` as we are not using it anymore